### PR TITLE
ES-524: remove all hardcoded values for insecureSkipVerify from the code

### PR DESCRIPTION
**Pull request by Qeas**

### CODE REVIEW

**Review by EvgeniiIvanov — APPROVED** [review]
LGTM

File: pkg/rest/client.go [hunk 1]
```diff
@@ -1,154 +0,0 @@
-package rest
```
Comment (goldranjith) [inline pkg/rest/client.go:1]: Please explain removing this file does not affect anything. wasn't this file used anywhere else?

**Comment by goldranjith** [comment]
@EvgeniiIvanov was functional test run and did the tests pass? pls update those results as well here.

**Comment by EvgeniiIvanov** [comment]
@goldranjith yes, it was checked. All works as expected.

File: pkg/rest/client.go (hunk 1 above)
Comment (Qeas) [reply pkg/rest/client.go:1]: This package was moved to a separate go lib here:  https://github.com/Nexenta/go-nexentastor

**Review by goldranjith — APPROVED** [review]
LBTM

**Pull request merged**